sympy/matrices/matrices.py

```diff
@@ -119,24 +119,24 @@ def entry(i,j):
 
     def _eval_berkowitz_vector(self):
         """ Run the Berkowitz algorithm and return a vector whose entries
-            are the coefficients of the characteristic polynomial of `self`.
+            are the coefficients of the characteristic polynomial of ``self``.
 
             Given N x N matrix, efficiently compute
-            coefficients of characteristic polynomials of 'self'
+            coefficients of characteristic polynomials of ``self``
             without division in the ground domain.
 
             This method is particularly useful for computing determinant,
-            principal minors and characteristic polynomial when 'self'
+            principal minors and characteristic polynomial when ``self``
             has complicated coefficients e.g. polynomials. Semi-direct
             usage of this algorithm is also important in computing
             efficiently sub-resultant PRS.
 
-            Assuming that M is a square matrix of dimension N x N and
-            I is N x N identity matrix, then the Berkowitz vector is
+            Assuming that ``M`` is a square matrix of dimension N x N and
+            ``I`` is N x N identity matrix, then the Berkowitz vector is
             an N x 1 vector whose entries are coefficients of the
             polynomial
 
-                           charpoly(M) = det(t*I - M)
+                           ``charpoly(M) = det(t*I - M)``
```

Comment:
we don't need double backticks here. Rather some `.. math ..` would be good. See as an example https://github.com/sympy/sympy/blob/master/sympy/concrete/products.py#L31-L33.